ebu_tt_live/strings.py

```diff
@@ -55,4 +55,8 @@
 DOC_SEQ_REQ_SEGMENT = gettext('{sequence_identifier}: requesting segment ({begin} - {end})')
 
 FS_DEFAULT_CLOCK_USED = gettext('Availability time not provided, creating default clock for sequence: {sequence_identifier}')
-FS_MISSING_AVAILABILITY = gettext('Could not resolve availability time for item in sequence: {sequence_identifier}. Item discarded.')
\ No newline at end of file
+FS_MISSING_AVAILABILITY = gettext('Could not resolve availability time for item in sequence: {sequence_identifier} after writing {file_path} - no manifest line written.')
+
+# CONFIG
+CFG_FILENAME_PATTERN = gettext('{sequence_identifier}_{counter}.xml')
+CFG_MESSAGE_PATTERN = gettext('{sequence_identifier}_msg_{counter}.xml')
```

Comment:
Arguably we should add a config string for the manifest filename too, but we don't allow it to be set anywhere, so it's currently hard-coded.